### PR TITLE
Add the contribution guide

### PR DESCRIPTION
**Pull request by NeilKleistGao**
Related Issue: #137 

- Add error message when TypeScript is not installed
- Add Getting Started Guide in `README.md`

### CODE REVIEW

**Comment by NeilKleistGao** [comment]
Tell me if more details need to be added. 😺

File: README.md
Comment (chengluyu) [inline README.md]: We also tested `v18` if I remembered correctly?

**Comment by LPTK** [comment]
Cool! You can also mention that individual tests can be run with `-z`. For example, `~mlscriptJVM/testOnly mlscript.DiffTests -- -z parser` will watch for file changes and continuously run all parser tests (those that have `"parser"` in their name).

File: ts2mls/js/src/main/scala/ts2mls/TSCompilerInfo.scala
Comment (chengluyu) [inline ts2mls/js/src/main/scala/ts2mls/TSCompilerInfo.scala:16]: Is this change related to this PR? Or did I miss something?

File: README.md
Comment (chengluyu) [inline README.md]:
```suggestion
### Running the web demo locally
```

File: README.md
Comment (chengluyu) [inline README.md]:
```suggestion
  // Allow overriding which specific tests to run, sometimes easier for development:
```

You forgot the leading spaces. 😛

**Review by chengluyu — CHANGES_REQUESTED** [review]

File: ts2mls/js/src/main/scala/ts2mls/TSCompilerInfo.scala
Comment (NeilKleistGao) [reply ts2mls/js/src/main/scala/ts2mls/TSCompilerInfo.scala:16]: Not strongly related. But as required I added some error information. Maybe I should change the title of this PR, or simply use another PR to submit this change.

Comment (chengluyu) [reply ts2mls/js/src/main/scala/ts2mls/TSCompilerInfo.scala:16]: I think it's fine.

File: README.md
Comment (NeilKleistGao) [reply README.md]: I copied this information from [getting-started.md](https://github.com/hkust-taco/mlstruct/blob/mlstruct/getting-started.md). My node's version is also `17.x.x` so I'm not sure. :joy:

Comment (LPTK) [reply README.md]: Just look at NodeTest.scala. I think it now supports it.

**Review by LPTK — APPROVED** [review]
LGTM, thanks! But before you merge (by squashing and cleaning up the commit message), please describe the `-z` way as the main way to run individual tests (put it first). The other way is more of an ugly hack.

File: README.md
Comment (NeilKleistGao) [reply README.md]: Yes indeed.

**Comment by NeilKleistGao** [comment]
Seems I can't merge before Luyu review again. 🤔

**Comment by chengluyu** [comment]
Wait a second. I have some changes.

**Review by chengluyu — APPROVED** [review]

**Pull request merged**
